tests/js/spec/views/performance/content.spec.jsx

```diff
@@ -11,9 +11,8 @@ import TeamStore from 'sentry/stores/teamStore';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 import PerformanceContent from 'sentry/views/performance/content';
 import {DEFAULT_MAX_DURATION} from 'sentry/views/performance/trends/utils';
-import {vitalAbbreviations} from 'sentry/views/performance/vitalDetail/utils';
 
-const FEATURES = ['transaction-event', 'performance-view'];
```

Comment:
it looks like this was an old feature flag that I couldn't find any references to in the codebase. I assumed it was safe to remove from here too